readme.md

```diff
@@ -83,25 +83,23 @@ Run CLI utility:
 php codecept.phar
 ```
 
-### Composer
-
-```
-php composer.phar require "codeception/codeception"
-```
-
-Read Also [Installation](http://codeception.com/install) | **[QuickStart](http://codeception.com/quickstart)**
+See also [Installation](http://codeception.com/install) | **[QuickStart](http://codeception.com/quickstart)**
 
 ## Getting Started
 
-If you successfully installed Codeception, run this command:
+After you successfully installed Codeception, run this command:
 
 ```
 codecept bootstrap
 ```
 
-this will create a default directory structure and default test suites
+This will create a default directory structure and default test suites.
+
+## Documentation
+
+[Documentation](http://codeception.com/docs/01-Introduction)
 
-See Documentation for more information.
+Documentation is included within the project. Look for it in the ['docs' directory](https://github.com/Codeception/Codeception/tree/master/docs).
```

Comment:
Yeah, OK. My main point: The "Documentation" link should lead to the actual documentation website - not to the docs directory.

Can you undo this part? Or do I have to create a separate commit? I'm no GitHub expert.....